scheduler/config-k8s.edn

```diff
@@ -19,11 +19,13 @@
  :compute-clusters [{:factory-fn cook.kubernetes.compute-cluster/factory-fn
                      :config {:compute-cluster-name "gke-1"
                               ;; Location of the kubernetes config file. Hardcoded to the location specified by bin/make-gke-test-cluster
-                              :config-file "../scheduler/.cook_kubeconfig_1"}}
+                              :config-file "../scheduler/.cook_kubeconfig_1"
+                              :blacklist-labels ["blacklist-nodes-with-this-label-key"]}}
```

Comment:
I think we should avoid "blacklist". How about "blocklist" or "deny-list" or "ignore"?

Also, maybe the word "node" should be in the config key, e.g. `:node-ignore-list`.

Reply:
Agreed. Changed.